hayeon/baekjoon/수학/4153-직각삼각형.js

```diff
@@ -0,0 +1,39 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  let result = "";
+
+  for (let i = 0; i < input.length - 1; i++) {
+    const [side1, side2, side3] = input[i]
+      .split(" ")
+      .map(Number)
+      .sort((a, b) => a - b);
+
+    const rightTriangle =
+      // Math.pow(side3, 2) === Math.pow(side1, 2) + Math.pow(side2, 2);
+      side3 ** 2 === side1 ** 2 + side2 ** 2;
+
+    result += rightTriangle ? `right\n` : `wrong\n`;
+  }
+
+  console.log(result);
```

Comment:
```suggestion
  let result = "";

  for (let i = 0; i < input.length - 1; i++) {
    const [side1, side2, side3] = input[i]
      .split(" ")
      .map(Number)
      .sort((a, b) => a - b);

    const isTriangle = side3 ** 2 === side1 ** 2 + side2 ** 2;

    result += isTriangle ? `right\n` : `wrong\n`;
  }

  console.log(result);
```
- 풀이 잘봤습니다 🙇‍♂️ boolean 형태라 is~~ 형태가 좀 더 명확할 것 같습니다 :)

Reply:
헉 그렇네요!!!!!!! 오우,,이런 디테일 미쳤숩니다 감사합니다!!!!!!👼🏻